src/plugins/intel_cpu/src/transformation_pipeline.cpp

```diff
@@ -611,7 +612,9 @@ void Transformations::MainSnippets(void) {
                                                            ov::is_type<const ov::op::v0::MatMul>(n) ||
                                                            ov::is_type<const ov::op::v1::Transpose>(n) ||
                                                            ov::is_type<const ov::op::v1::Broadcast>(n) ||
-                                                           ov::is_type<const ov::op::v3::Broadcast>(n));
+                                                           ov::is_type<const ov::op::v3::Broadcast>(n) ||
+                                                           (ov::is_type<const ngraph::op::FakeQuantize>(n) &&
+                                                           !ngraph::snippets::pass::FakeQuantizeDecomposition::checkConstants(n)));
```

Comment:
I think that you've added this condition because of bug from ticket 100808.
I have the fix 9c467ea313c117a3175b8afa179340679d560797 for that. Could you please confirm if it's this bug or clarify what is the reason of condition addition?

Reply:
Correct, it's quick fix to avoid fail in CPU for different constant shapes. Looks like it's connected with 100808 issue.

Reply:
FYI: The [PR](https://github.com/openvinotoolkit/openvino/pull/15837) with the fix

Reply:
We can remove it due PR has been merged. Just rebase on the latest master please